trin-history/src/storage.rs

```diff
@@ -218,14 +205,13 @@ impl HistoryStorage {
 
         // Store the data in db
         let content_key: Vec<u8> = key.clone().into();
-        // store content key w/o the 0x prefix
-        let content_key = hex_encode(content_key).trim_start_matches("0x").to_string();
         let value_size = value.len() as u64;
         match self.db_insert(&content_id, &content_key, value) {
             Ok(result) => {
                 // Insertion successful, increase total network storage count
                 if result == 1 {
-                    self.storage_occupied_in_bytes += value_size;
+                    // adding 32 bytes for content_id and 32 for content_key
+                    self.storage_occupied_in_bytes += value_size + 64;
```

Comment:
Maybe create a function in `trin-storage/src/utils.rs` for calculating the content_size. That way it can be documented once and we don't have to add constants all over the code.

It might be overkill from function functionality, but I think it improves readability.

Reply:
We are most likely going to replace this with your code in a week or 2 so I think that util function would be overkill as we will be removing it soon enough. I will make a constant for it for the time being though